specification/dimensions/chargetype.md

```diff
@@ -0,0 +1,38 @@
+# Charge type
+
+A charge type indicates whether the record represents a prepayment (e.g., up-front or recurring fee), cost of usage that already occurred, an after-the-fact adjustment (e.g., credits), or taxes. The charge type is commonly used to identify and analyze prepaid purchases or usage-based charges.
```

Comment:
Technically, that would be a grammatically incorrect use of casing, since it's not a proper noun. I can confirm with my content team, but I suspect they'll refer back to a style guide. Have we chosen one?

Reply:
This is the only one we've discussed previously: https://ieeecs-media.computer.org/assets/pdf/2016CSStyleGuide.pdf 

Tell me why this wouldn't be a proper noun again? (not an expert but I thought this could be a 'Thing')

Reply:
A charge type is not specific to a single (proper noun) charge, but a class of charges (common noun). I think for the descriptive prose we adhere to normal English grammar (in this case lowercase). For the normative prose we use capitals and prefer the column name vs the display name.

Reply:
Proper nouns can serve as the name of a specific person, place, or thing. Since there isn't one and only one charge type, that doesn't apply here. I'm not an English major, but that's my understanding anyway.

Reply:
sounds good. I'll update the rest.